tsfresh/feature_extraction/settings.py

```diff
@@ -117,7 +119,7 @@ def __init__(self):
             "ar_coefficient": [{"coeff": coeff, "k": k} for coeff in range(5) for k in [10]],
             "mean_abs_change_quantiles": [{"ql": ql, "qh": qh}
                                           for ql in [0., .2, .4, .6, .8] for qh in [.2, .4, .6, .8, 1.]],
-            "fft_coefficient": [{"coeff": coeff} for coeff in range(0, 10)],
+            "fft_coefficient": [{"coeff": k, "attr": a} for a, k in product(["real", "imag"], range(100))],
```

Comment:
Cool, that means we could derive the number of coefficients to return as features from the length of the time series. I think that would be better than to return lots of `NaN` or skipping possibly meaningful coefficients. But of course it would be problematic to vary the features depending on the length of the time series, as the length could differ between samples. This would be especially problematic with regards to train / test splits.